CIP-DappConnector/CIP-dapp-connector.md

```diff
@@ -82,6 +84,7 @@ APIError {
 * InvalidRequest - Inputs do not conform to this spec or are otherwise invalid.
 * InternalError - An error occurred during execution of this API call.
 * Refused - The request was refused due to lack of access - e.g. wallet disconnects.
+* AccountChange - The account has changed. The dApp should call `wallet.enable()` to reestablish connection to the new account. The wallet should not ask for confirmation as the user was the one who initiated the account change in the first place.
```

Comment:
We had no other way of knowing with the events API removed so I added this in. It's not ideal control flow but it should suffice until we have that implemented.